optimum/exporters/ipex/model_patcher.py

```diff
@@ -73,6 +79,20 @@ def _patch_llama_model(model):
     return model
 
 
+def _patch_falcon_model(model):
```

Comment:
could you add type hints + docstring for all ?

Reply:
Done.

Reply:
thanks for adding that looks great